libs/shared/providers/src/prices/constants.ts

```diff
@@ -176,6 +176,11 @@ export const priceOptions: Partial<Record<SupportedTokenPrice, PriceOption>> = {
     id: 'stETH_USD',
     dependsOn: ['stETH_ETH', 'ETH_USD'],
   },
+  wOETH_ETH: {
+    type: 'derived',
+    id: 'wOETH_ETH',
+    dependsOn: ['wOETH_OETH'], // In one direction this is correct, in the other direction it is off by ~.1% (?)
```

Comment:
The problem I had was that we don't have an OETH_ETH and I thought that since this is only off by ~.1% in one of the potential directions that it might suffice.

Reply:
OETH_ETH is a bit more complex, for now it goes through curve, you can retrieve the logic from the `estimateAmount` call [here](https://github.com/OriginProtocol/origin-defi/blob/00c2e72bc9dabf3e6bf3ceb9a533e61de323a4b2/libs/oeth/swap/src/actions/swapCurve/index.ts#L36). Maybe there's a better way...